tools/api-collection/cloudflare-collection.json

```diff
@@ -0,0 +1,208 @@
+{
+  "_type": "export",
+  "__export_format": 4,
+  "__export_date": "2024-04-09T10:20:24.174Z",
+  "__export_source": "insomnia.desktop.app:v2023.5.7-adobe",
```

Comment:
Nit: Are these necessary field ?

Reply:
I just exported the collection from the postbuster with default settings, it should be okay as its working fine on both postbuster and postman